source/Halibut/Transport/SecureListener.cs

```diff
@@ -164,7 +165,7 @@ void WaitForPendingConnectionOrCancellation()
                     catch (Exception ex)
                     {
                         numberOfFailedAttemptsInRow++;
-                        log.WriteException(EventType.ErrorInInitialisation, "Error accepting TCP client", ex);
+                        log.WriteException(EventType.ErrorInInitialisation, "Error accepting TCP client: {0}", ex, client?.Client.RemoteEndPoint);
```

Comment:
Client.RemoteEndPoint can throw exceptions so do you think it would be safer to call an extension method or similar that wraps the call in a try catch so we guarantee we don't error trying to log an error 
![image](https://github.com/OctopusDeploy/Halibut/assets/86938706/5abcef74-ef17-4a99-9664-f14e75c6c852)

Reply:
I don't know if it would ever throw in the situation we are using it, but just incase

Reply:
Kind of a on th lines of Halibut.Transport.Proxy.Utils.GetHost and GetPort